### PR TITLE
fix(textarea): native textarea inherits max-width and max-height

### PR DESCRIPTION
**Pull request by liamdebeasi**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Build (`npm run build`) was run locally and any changes were pushed
- [x] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Pull request type

 

 

Please check the type of change your PR introduces:
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?

Issue Number: fixes https://github.com/ionic-team/ionic/issues/18543

## What is the new behavior?

- Uses inheritance for allow for max-width/max-height instead of exposing a CSS variable
- alternative to https://github.com/ionic-team/ionic/pull/19617

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: core/src/components/textarea/textarea.scss [hunk 1]
```diff
@@ -74,6 +74,14 @@
 // Native Textarea
 // --------------------------------------------------
 
+.textarea-wrapper {
+
```
Comment (brandyscarney) [inline core/src/components/textarea/textarea.scss:78]:
```suggestion
```

File: core/src/components/textarea/textarea.scss [hunk 2]
```diff
@@ -49,6 +48,7 @@
   white-space: pre-wrap;
 
   z-index: $z-index-item-input;
+  box-sizing: border-box;
```
Comment (brandyscarney) [inline core/src/components/textarea/textarea.scss:51]:
```suggestion
  z-index: $z-index-item-input;

  box-sizing: border-box;
```

File: core/src/components/textarea/textarea.scss (hunk 1 above)
Comment (liamdebeasi) [reply core/src/components/textarea/textarea.scss:78]: 😂

Comment (brandyscarney) [reply core/src/components/textarea/textarea.scss]: Sorry 😂

**Pull request merged**